pages/posts/roadmap-2024.md

```diff
@@ -0,0 +1,228 @@
+
+Specifically: 
+
+- The History sub-protocol enables Ethereum execution clients to expire historical data via EIP-4444, safe in the knowledge that the data is available on request from the Portal Network. 
+- The State sub-protocol gives light clients access to Ethereum stata data, exposing methods such as `eth_getProof` that eliminate the need for trusted third party RPC providers. 
+- The Beacon sub-protocol gives light clients access to the latest block headers so that they can be confident they are following the canonical Ethereum chain.
+
+Shipping these features will enable ***major upgrades for the Ethereum ecosystem as a whole***. These features have been in development for several years and have been through several rounds of refinement and course correction. Now Portal Network is on the brink of shipping these sub-networks into production. 
+
+This is why ***2024 is a set to be a standout year for Portal Network.***
+
+Longer term, there are proposals for more sub-protocols that serve granular transaction data, blob data, proofs etc, but these are out of scope for 2024.
+
+
+## High level goals
+
+### 2024 
+
+- **Objective 1: Ship a viable solution for historical data storage**
+    - Key result 1:
+        -  Portal History subprotocol is fully featured according to the canonical specification and there is feature parity across all three Portal clients. 
+    - Key result 2:
+        - \>=1 execution client is using Portal History network to support EIP-4444
+    - Key result 3:
+        - History network never drops below 99% score on Glados
+    - Key result 4:
+        - EIP-4444 is shipped for execution clients, with Portal Network named as the default long term data availability solution.
+
+
+- **Objective 2: Ship a viable solution for decentralized access to Beacon block headers**
+    -  Key result 1:
+        -  Portal Beacon subprotocol is fully featured according to the canonical specification and there is feature parity across all three Portal clients. 
+    -  Key result 2:
+        -  Beacon network never drops below 99% score on Glados
+    -  Key result 3:
+        - \>=1 application or light client using Portal Network to follow the head of the Beacon chain
+    - Key result 4:
+        - \>=1 execution client using Portal Beacon network to accelerate their syncing
+
+
+- **Objective 3: Ship a viable solution for decentralized access to state data**:
+    - Key result 1:
+        -  Portal State subprotocol is fully featured according to the canonical specification and there is feature parity across all three Portal clients. 
+    -  Key result 2:
+        -  State network never drops below 99% score on Glados
+    -  Key result 3:
+        - \>=1 application or client using Portal State network to request `eth_getProof` data
+
+- **Objective 4: Develop and maintain reliable tooling**
+    - Key result 1: 
+        - Glados maintains 99% uptime and is able to handle high volume of traffic (up to 10^5 visits/day)
```

Comment:
The above objectives already imply it, but maybe we want to explicitly add: 
`- Glados provides network audit statistics for the history, beacon, and state networks`